proto/api.proto

```diff
@@ -363,13 +369,13 @@ service Api {
     };
   }
 
-  rpc GetTickersStream(GetTickersRequest) returns (stream GetTickersStreamResponse) {
+  rpc GetMarketDepthsStream(GetMarketDepthsRequest) returns (stream GetMarketDepthsStreamResponse) {
```

Comment:
looks like we're a bit inconsistent with this, but I think we should try to include `StreamRequest` in the parameter name, e.g. `GetMarketDepthsStreamRequest`

Reply:
not really, the requests don't have "stream" in other endpoints.